### DISCUSSION TITLE
Official ansible collection for teleport?

### DISCUSSION THREAD

**Discussion by gardar** [gravitational/teleport#38740] [Ideas]
Are there any plans of creating a official ansible collection for teleport? Or perhaps adopting one from the community?

I see huge potential with a ansible collection that could include the following:
* Ansible role for teleport server installs and HA setups
* Ansible role for the proxy
* Ansible role for client agent / agentless setup
* Ansible modules for configuring teleport as a alternative to the the terraform resources
* Ansible inventory plugin to use teleport as a inventory for ansible

**Comment by stevenGravy** [comment]
Hi @gardar , that has come up for discussion and there is likely a github issue on it. For now Teleport has not put out specific plans for a ansible collection like this. For self-hosting clients we see a diverse set of usage for installation such as Kubernetes which a operator is provided. That may be revisited and thanks for listing out the asks.

**Reply by gardar** [reply]
Thanks for the reply.
Would there be any interest in adopting a ansible collection for teleport provided by the community as a official teleport project? 
Or is there perhaps some github org for teleport-community for community maintained projects?
